### PR TITLE
Update samsungtv integration with new Remote entity

### PR DESCRIPTION
**Pull request by peitschie**
## Proposed change

This is the documentation update for https://github.com/home-assistant/core/pull/83439

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/83439
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/samsungtv.markdown [hunk 1]
```diff
@@ -94,6 +96,53 @@ media_content_type: channel
 It's possible to switch between the 2 sources `TV` and `HDMI`.
 Some older models also expose the installed applications through the WebSocket, in which case the source list is adjusted accordingly.
 
+### Remote
+
+The integration supports `remote` platform. The remote allows you to send key commands to your TV with the `remote.send_command` service.
+
+The exact supported keys varies between TV models. See [this link](https://github.com/jaruba/ha-samsungtv-tizen/blob/master/Key_codes.md) for possible keys.
```
Comment (frenck) [inline source/_integrations/samsungtv.markdown]:
```suggestion
The exact supported keys vary between TV models. See [this link](https://github.com/jaruba/ha-samsungtv-tizen/blob/master/Key_codes.md) for possible keys.
```

Comment (frenck) [reply source/_integrations/samsungtv.markdown]: I'm not sure why we have to link into an external GitHub page?
We already document it below, right?

File: source/_integrations/samsungtv.markdown [hunk 2]
```diff
@@ -94,6 +96,53 @@ media_content_type: channel
...
+- KEY_0
+- KEY_1
+- KEY_2
+- KEY_3
+- KEY_4
+- KEY_5
+- KEY_6
+- KEY_7
+- KEY_8
+- KEY_9
```
Comment (frenck) [inline source/_integrations/samsungtv.markdown]:
```suggestion
- `KEY_UP`
- `KEY_DOWN`
- `KEY_LEFT`
- `KEY_RIGHT`
- `KEY_ENTER`
- `KEY_RETURN`
- `KEY_HOME`
- `KEY_REWIND`
- `KEY_FORWARD`
- `KEY_ACTIONMENU`
- `KEY_0`
- `KEY_1`
- `KEY_2`
- `KEY_3`
- `KEY_4`
- `KEY_5`
- `KEY_6`
- `KEY_7`
- `KEY_8`
- `KEY_9`
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/samsungtv.markdown (hunk 1 above)
Comment (peitschie) [reply source/_integrations/samsungtv.markdown]: This isn't the full key set by any stretch. The full list is more than 200 keys 🙂

Comment (frenck) [reply source/_integrations/samsungtv.markdown]: Isn't that an issue already? You now link to an external page, which is in GitHub, that imposes the same full list? How is that better?

Comment (peitschie) [reply source/_integrations/samsungtv.markdown]: Sorry, I'm not following the issue here.
I've not duplicated the full list, I've merely included a very small sample.

Are you suggesting we shouldn't link to the complete list of key codes here? Or are you suggesting we copy all 200+ inline here? 😅

Comment (frenck) [reply source/_integrations/samsungtv.markdown]: IMHO both are bad? 

We can just add all of them and put them in a details block so their are only shown when a user expands it.

../Frenck

Comment (peitschie) [reply source/_integrations/samsungtv.markdown]: Done!

File: source/_integrations/samsungtv.markdown [hunk 3]
```diff
@@ -94,6 +96,377 @@ media_content_type: channel
 It's possible to switch between the 2 sources `TV` and `HDMI`.
 Some older models also expose the installed applications through the WebSocket, in which case the source list is adjusted accordingly.
 
+### Remote
+
+The integration supports `remote` platform. The remote allows you to send key commands to your TV with the `remote.send_command` service. The supported keys vary between TV models.
```
Comment (MartinHjelmare) [inline source/_integrations/samsungtv.markdown]:
```suggestion
The integration supports the `remote` platform. The remote allows you to send key commands to your TV with the `remote.send_command` service. The supported keys vary between TV models.
```

**Review by frenck — APPROVED** [review]
Thanks, @peitschie 👍 

../Frenck

**Pull request merged**